binarySearch/binarySearch.js

```diff
@@ -10,6 +10,76 @@
  *	* console.log(index); // 1
 **/
 
+
+/* EVERYTHING stolen from here ,just wanted to submit a working something
```

Comment:
Ah! I missed this line when I first saw your solution and thought it looked different than your typical style.. no es6 etc. Also on lines 18-20 I see that as nothing more than lazy programming. You can define variables that way but it just tells me the person isn't willing to write out `var` or `let` multiple times which makes the code more readable. Anyway, feel free to take your time on these challenges. Don't feel pressured to get a working solution the day after we present them. Maybe take the time during the solution lecture to understand the concepts and go back and take a crack at it later. You're doing great, and don't need to go looking for solutions to these problems online if you're struggling to come up with one on your own.